clusterloader2/pkg/modifier/modifier.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package modifier
+
+import (
+	"k8s.io/perf-tests/clusterloader2/api"
+	"k8s.io/perf-tests/clusterloader2/pkg/flags"
+)
+
+type Modifier interface {
+	ChangeTest(*api.Config)
+}
+
+var (
+	skipSteps []string
```

Comment:
I'm not a fan of global variables.

I think that creating `Config` type and adding it as a field to cclusterLoaderConfig would be cleaner.

Reply:
Make sense, tried to follow convection - some flags are using global variables.

There is mix in the code with this approach. Added ModifierConfig with the same approach as PrometheusConfig - PTAL if this makes sense.